src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -510,6 +510,11 @@ void LogError(IRestResponse response)
             LogTo.Error(exception, errorMessage);
         }
 
+        string CleanUp(string content)
+        {
+            return content.StartsWith(byteOrderMarkUtf8) ? content.Remove(0, byteOrderMarkUtf8.Length) : content;
+        }
+
```

Comment:
Only a minor cosmetic point, but it would have been nicer if the `CleanupBodyString` method above would have been replaced inline, conserving the method name, to isolate the changes in the diff.